### DISCUSSION TITLE
How to disable navigation label button?

### DISCUSSION THREAD

**Discussion by kharithomas** [wojtekmaj/react-calendar#717] [Q&A] [answered]
Is there a way we can disable the navigation label button that stores the month and year (see below next to red arrow)?

My calendar doesn't allow past dates to be selected so I'd like to remove the ability to click the navigation also.

![Group 160](https://user-images.githubusercontent.com/55037984/208222178-1d085ba1-cf9e-417a-8d45-4800ebc01f24.png)

**Comment by dawid-grabowski** [comment, marked as answer]
To disable clicking on month label use `minDetail="month"`. It should do the thing.

**Reply by AguChaves92** [reply]
can you disable clicking on weekdays tiles (M,T,W, etc)?

**Comment by IgorLab19k** [comment]
Hey bro, how did you made week days start from Sunday ? :)

**Reply by wojtekmaj** [reply]
Check `calendarType` and `locale` in documentation.

**Reply by kharithomas** [reply]
@IgorLab19k Yes as mentioned by @wojtekmaj . I have my calendar component set like:

`<Calendar calendarType="US" //... rest of props />`

**Reply by SpaceHolderdJs** [reply]
Perfect, thanks 🙂
